Sources/KlaviyoSwift/KlaviyoEnvironment.swift

```diff
@@ -23,20 +25,45 @@ let decoder = {
     return decoder
 }()
 
+private let reachabilityService = Reachability(hostname: "a.klaviyo.com")
+
 struct KlaviyoEnvironment {
     var archiverClient: ArchiverClient
     var fileClient: FileClient
     var data: (URL) throws -> Data
     var logger: LoggerClient
     var analytics: AnalyticsEnvironment
     var getUserDefaultString: (String) -> String?
-    static let production = KlaviyoEnvironment(
+    var appLifeCycle: AppLifeCycleEvents
+    var notificationCenterPublisher: (NSNotification.Name) -> AnyPublisher<Notification, Never>
+    var legacyIDFV: () -> String
```

Comment:
idfv?

Reply:
Yes. We moved from this to a stored uuid. Maybe I'll change this to legacyIdentifier since it's not quite an IDFV. It has a "iOS:" prefixed to it.